README.md

```diff
@@ -52,15 +52,41 @@ bookshelf.Model.extend({
 
 ## Contributing
 
-Feel free to fork this repository and submit pull requests. To run the tests, duplicate the *test/knexfile.js.dist* file, update it to your needs and run:
+Contributions are welcome and greatly appreciated, so feel free to fork this repository and submit pull requests.  
+
+- **bookshelf-json-columns** supports PostgreSQL and SQLite3.
```

Comment:
Replace by a single paragraph:

**bookshelf-json-columns** supports PostgreSQL and SQLite3. You can find test suites for each of these database engines in the *test/postgres* and *test/sqlite* folders.